src/query/useFriendBot.ts

```diff
@@ -20,7 +20,7 @@ export const useFriendBot = ({
       : "https://friendbot.stellar.org";
 
   const query = useQuery({
-    queryKey: ["friendBot", key],
+    queryKey: ["friendBot", publicKey, key],
```

Comment:
We need this to be unique to fetch account info on the Saved Keypairs page.

Reply:
thank you